src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -35,6 +35,8 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction Average = _library.Append(new AverageFunction());
         public static readonly TexlFunction AverageT = _library.Append(new AverageTableFunction());
         public static readonly TexlFunction Blank = _library.Append(new BlankFunction());
+        public static readonly TexlFunction Boolean = _library.Append(new BooleanFunction());
+        public static readonly TexlFunction Boolean_UO = _library.Append(new BooleanFunction_UO());
```

Comment:
Just confirming if we want these to show up in Canvas yet? 
(see [https://msazure.visualstudio.com/OneAgile/_wiki/wikis/OneAgile.wiki/216333/Merging-into-PAClient](https://msazure.visualstudio.com/OneAgile/_wiki/wikis/OneAgile.wiki/216333/Merging-into-PAClient))

Reply:
For Boolean, yes, we can add the TS implementation when we upgrade the Nuget package version. I moved the _UO function down to the bottom with the rest. It is no longer being added to the library by default.

Reply:
" we can add the TS implementation when we upgrade the Nuget package version" 
We should either have that PR ready to go, or keep Boolean off the list until it is.  Else whoever upgrades the version number is going to be hit with this.